### PR TITLE
Swagger changes to support Vertica V2 in ADF

### PR DESCRIPTION
**Pull request by kanikagupta11**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/LinkedService.json [hunk 1]
```diff
@@ -6161,11 +6161,36 @@
...
+          "description": "Server name for connection. Type: string."
+        },
+        "port": {
+          "type": "object",
+          "x-ms-format": "dfe-int",
+          "description": "The port for the connection. Type: integer."
+        },
+        "username": {
+          "type": "object",
+          "x-ms-format": "dfe-string",
```
Comment (ruowan) [inline specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/LinkedService.json:6185]: Please remove the x-ms-format for the synapse swagger.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 2]
```diff
@@ -6710,8 +6710,27 @@
       "properties": {
         "connectionString": {
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
+          "type": "object"
+        },
+        "server": {
           "type": "object",
-          "x-ms-format": "dfe-string"
```
Comment (ruowan) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json:6714]: Why the dfe-string remvoed from the connectionString. Please reverted it back if by mistake.

**Review by Frey-Wang — APPROVED** [review]

**Pull request merged**